modules/oncodriveclustl/1.0.1/oncodriveclustl.smk

```diff
@@ -0,0 +1,428 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  N/A
+# Module Author:    Manuela Cruz
+# Contributors:     N/A
+
+
+import oncopipe as op
+import os
+import pandas as pd
+import hashlib
+
+### stuff that is not supposed to be here:
+configfile: "src/lcr-modules/modules/oncodriveclustl/1.0.1/config/default.yaml"
+
+config["tool_names"] = "oncodriveclustl"
+config["pipeline_name"] = "oncodriveclustl"
+
+include: "header_2.1.smk"
+
+config["lcr-modules"]["oncodriveclustl"]["dirs"]["_parent"] = config['lcr-modules']['_shared']['root_output_dir'] + "oncodriveclustl_module_test/"
+
+#### FOR CHECKING MODULE RUNS:
+RUN_SAMPLES = PIPELINE_SAMPLES["SAMPLES"]["oncodriveclustl"]
+#### add this to code below? kinda necessary to run?
+####
```

Comment:
This stuff is all GAMBL-specific and really belongs in the Snakefile that you would use to launch this module. In that Snakefile you would store the samples table in the config as follows: 
```
config["lcr-modules"]["oncodriveclustl"]["samples"] = PIPELINE_SAMPLES["SAMPLES"]["oncodriveclustl"]
```

Then here in the module you can create the `RUN_SAMPLES` variable like this: 
```
RUN_SAMPLES = CFG["samples']
```